src/dotdom.js

```diff
@@ -272,5 +260,4 @@ module.exports = window;
         )                                                             // tag named as the property requested.
```

Comment:
Don't forget to change `createElement.bind(global, tagName)` to `createElement.bind(window, tagName)`